app/models/item_device.py

```diff
@@ -35,10 +35,18 @@ def __str__(self):
         else:
             return '-'
 
+    """
+    :return: item device dict for csv
+    :rtype: dict
+
+    for the device without its type, set it as MISCELLANEOUS
+    """
     def csv_dict(self):
         return {
-            "Category - Item Device Type": self.dtype.device_type,
-            "Type - Item Device Type": self.dtype.category,
+            "Category - Item Device Type": self.dtype.device_type
+                if self.dtype is not None else ITEM_MAP.get("")["device_type"],
```

Comment:
Please make sure that the exported names of default `ITEM_MAP.get("")["device_type"]` matches that of the `ItemCategoryEnum` display name. Please see https://github.com/CodeTheChangeUBC/reBOOT/blob/master/app/models/item_device_type.py